File: Learn the Basic/Basic of programming/Switch/C++/first.cpp
#include<iostream>
using namespace std;
int main(){
    int day;
    cout<<"Enter Day from 0-7: ";
    cin>>day;

    switch (day)
    {
    case 1:
        cout<<"Monday";
        break;
    case 2:
        cout<<"Tuesday";
        break;
    case 3:
        cout<<"wednessday";
        break;
    
        case 4:
        cout<<"thrusday";
        break;

    case 5:
        cout<<"Friday";
        break;
    case 6:
        cout<<"saturday";
        break;
    
        case 7:
        cout<<"sunday";
        break;
    default :
        cout<<"invalid";
    }
}

File: Learn the Basic/Basic of programming/Switch/python/first.py
day=int(input("Enter the day (1-7): "))
match day:
    case 1:
        print("Monday")
    case 2:
        print("Tuesday")
    case 3:
        print("Wednesday")
    case 4:
        print("Thrusday")
    case 5:
        print("Friday")
    case 6:
        print("Saturday")
    case 7:
        print("Sunday")
    case _:
        print("Invalid ")
        